<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEdit</class>
 <widget class="QDialog" name="DialogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabAdd">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabScientist">
      <attribute name="title">
       <string>Scientist</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblName">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblGender">
           <property name="text">
            <string>Gender:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtGender"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblBirth">
           <property name="text">
            <string>Year of birth:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtBirth"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblDeath">
           <property name="text">
            <string>Year of death:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtDeath"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Connect scientist to computers</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Computers connnected</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Computers NOT connected</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QListView" name="lstConnComps"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QPushButton" name="btnAddSciConn">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDelSciConn">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListView" name="lstNotConnComps"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabComputer">
      <attribute name="title">
       <string>Computer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="lblCompName">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtCompName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lblType">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtType"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lblYearBuilt">
           <property name="text">
            <string>Year built:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtYearBuilt">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lblBuilt">
           <property name="text">
            <string>Has it been built?</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioBuiltYes">
           <property name="text">
            <string>Yes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioBuiltNo">
           <property name="text">
            <string>No</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="grpCompConns">
         <property name="title">
          <string>Connect computer to scientists</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Scientists Connected</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Scientists NOT connected</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QListView" name="lstConnSci"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="btnAddConn">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDelConn">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListView" name="lstNotConnSci"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="btnBoxComputer">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
